### PR TITLE
Remove use of Span ctor(T[], int) which no longer exists

### PR DESCRIPTION
**Pull request by ahsonkhan**
Also remove duplicate test cases

cc @KrzysztofCwalina

### CODE REVIEW

**Comment by ahsonkhan** [comment]
@KrzysztofCwalina, this is a fairly benign PR. Should I go ahead and merge it?

File: tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserIntegerTests.cs [hunk 1]
```diff
@@ -685,15 +685,10 @@ public unsafe void ParseUInt64Hex(string text, bool expectSuccess, UInt64 expect
         [InlineData("-0128", true, -128, 5)] // min
         [InlineData("0128", false, 0, 0)] // positive overflow test
         [InlineData("-0129", false, 0, 0)] // negative overflow test
-        [InlineData("111", true, 111, 3)]
```
Comment (KrzysztofCwalina) [inline tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserIntegerTests.cs]: Why are these changes here?

**Comment by KrzysztofCwalina** [comment]
looks good, if the tests are really duplicate.

File: tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserIntegerTests.cs (hunk 1 above)
Comment (ahsonkhan) [reply tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserIntegerTests.cs]: Yes, these test cases are duplicate. It was observed when VS test discovery stated explicitly that the tests have already been added to the test window and hence ignoring it.

Line 688 is: `[InlineData("111", true, 111, 3)]`
Which is the same as line 649 - https://github.com/dotnet/corefxlab/pull/1725/files/d6f7d417dcd26ee92a6f13795ee58273fe24bf26#diff-de2a92cba24dadec8a492537ca278d1bL649
